pkg/controller/registry/resolver/rbac.go

```diff
@@ -90,6 +90,21 @@ func RBACForClusterServiceVersion(csv *v1alpha1.ClusterServiceVersion) (map[stri
 			permissions[permission.ServiceAccountName] = NewOperatorPermissions(serviceAccount)
 		}
 
+		operatorConditionRBAC := []rbacv1.PolicyRule{
+			{
+				Verbs:         []string{"get"},
```

Comment:
Is it reasonable for an operator to update the spec or change the metadata (e.g. annotations, labels)?

Reply:
At this time no, we can always expand these privileges in the future